hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/ResourceVector.java

```diff
@@ -88,6 +89,15 @@ public void subtract(ResourceVector otherResourceVector) {
     }
   }
 
+  /**
+   * Subtracts the given resource by the specified value.
+   * @param resourceName name of the resource
+   * @param value value to be subtracted from the resource's current value
+   */
+  public void subtract(String resourceName, float value) {
```

Comment:
Nit: I would call this decrement, subtract suggests that you subtract another ResourceVector from this.

Reply:
Agreed, subtract would ideally return a new ResourceVector, fixed.